client/lib/user/shared-utils/filter-user-object.js

```diff
@@ -21,6 +21,7 @@ const allowedKeys = [
 	'logout_URL',
 	'primary_blog',
 	'primary_blog_is_jetpack',
+	'has_jetpack_partner_access',
```

Comment:
I think we will have to rename this property to something more specific since we want to detect agencies and not every type of partner here. For example, hosting companies have access to the Licensing Portal but won't have (for now at least) access to the dashboard, so we need to differentiate between the two.

Reply:
That is a good point, I will have to rethink some things. The boolean is intended to be generic enough to gate all requests to the partner API so it shouldn't be agencies only, but it is incorrect to use this boolean to determine if the partner is an agency.